helpdesk/email.py

```diff
@@ -56,11 +57,16 @@
 HTML_EMAIL_ATTACHMENT_FILENAME = _("email_html_body.html")
 
 
-def process_email(quiet=False):
+def process_email(quiet: bool = False, debug_to_stdout: bool = False):
+    if debug_to_stdout:
+        print("Extracting email into queues...")
+    q: Queue()  # Typing ahead of time for loop to make it more useful in an IDE
     for q in Queue.objects.filter(
             email_box_type__isnull=False,
             allow_email_submission=True):
-
+        log_msg = f"Processing queue: {q.slug} Email address: {q.email_address}..."
+        if debug_to_stdout:
+            print(log_msg)
         logger = logging.getLogger('django.helpdesk.queue.' + q.slug)
```

Comment:
I would prefer to configure this logger here to output to stdout if the flag is set, then duplicate the lines.

Reply:
I could but from a code perspective this is much cleaner as the debug_to_stdout flag is specifically only for manually running the 
The same arguments apply here as for the earlier comment regarding why this is treated separately.